### PR TITLE
trulens integration

### PR DESCRIPTION
**Pull request by joshreini1**
# Description

Add TruLens integration to the docs!

- [X] This change requires a documentation update

### CODE REVIEW

File: docs/how_to/integrations/trulens.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Evaluating and Tracking with TruLens
+
+This page covers how to use [TruLens](https://trulens.org) to evaluate and track LLM apps built on Llama-Index.
+
+## What is TruLens?
+
+TruLens is an [opensource](https://github.com/truera/trulens) package that provides instrumentation and evaluation tools for large language model (LLM) based applications. This includes feedback function evaluations of relevance, sentiment and more + in-depth tracing including cost and latency.
+
+
+As you iterate new versions of your LLM application, you can compare their performance across all of the different quality metrics you've set up. You'll also be able to view evaluations at a record level, and explore the app metadata for each record.
```
Comment (jerryjliu) [inline docs/how_to/integrations/trulens.md:10]: *as you iterate on

File: docs/how_to/integrations/trulens.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Evaluating and Tracking with TruLens
+
+This page covers how to use [TruLens](https://trulens.org) to evaluate and track LLM apps built on Llama-Index.
+
+## What is TruLens?
+
+TruLens is an [opensource](https://github.com/truera/trulens) package that provides instrumentation and evaluation tools for large language model (LLM) based applications. This includes feedback function evaluations of relevance, sentiment and more + in-depth tracing including cost and latency.
```
Comment (jerryjliu) [inline docs/how_to/integrations/trulens.md:7]: doesn't have to be this PR - an image might be nice

**Review by jerryjliu — COMMENTED** [review]
@joshreini1 could you add a link to `docs/how_to/integrations.rst`? we'll reorganize the integrations docs later

**Comment by joshreini1** [comment]
Updated - thanks for the review @jerryjliu !

**Review by jerryjliu — APPROVED** [review]

**Pull request merged**